### ISSUE TITLE
Welsh {train/dev/test} splits not working

### ISSUE THREAD

**Issue by JRMeyer** [mozilla/CorporaCreator#55]
the number of files in train + test + dev != valid
-----------------------------------

This behavior was validated by two people on two different machines.

Also, for Kyrgyz the numbers are also off, but less so

```
josh@carbon:~/git/CorporaCreator/corpora/cy$ wc -l *
    485 dev.tsv
    669 invalid.tsv
    305 other.tsv
    485 test.tsv
    500 train.tsv
  18420 valid.tsv
  20864 total
```

**Comment by simnotes** [comment]
I can say the same for the german language corpus. The "problem" is not a real problem. Let me explain:
you have 18420 valid sentences, but all of those sentences occur multiple times in the same dataset. Since you probable did not use the "-s" parameter witch CorpusCreator, every sentence available will only occur just once across all datasets, since the default-value for "-s" parameter is "1".
In other words: you only have 485 + 485 + 500 unique sentences in valid.tsv recorded by multiple people.

**Comment by kdavis-mozilla** [comment]
What does "not working" mean?
